<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardPreviewWidget</class>
 <widget class="QWidget" name="CardPreviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="frame_options_top">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinbox_zoom">
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_zoom">
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CardPreview" name="graphicsView">
     <property name="renderHints">
      <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
     <property name="optimizationFlags">
      <set>QGraphicsView::DontAdjustForAntialiasing</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_options">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_grid">
        <property name="text">
         <string>Grid</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_debug">
        <property name="text">
         <string>Draw Debug Info</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>314</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_zoom_50">
        <property name="text">
         <string notr="true">50%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_zoom_100">
        <property name="text">
         <string notr="true">100%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fit_view">
        <property name="text">
         <string>Fit in View</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CardPreview</class>
   <extends>QGraphicsView</extends>
   <header>cardpreviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
